src/app/core/store/organizations/organizations.effects.ts

```diff
@@ -66,14 +68,17 @@ export class OrganizationsEffects {
 
   @Effect()
   public update$: Observable<Action> = this.actions$.ofType<OrganizationsAction.Update>(OrganizationsActionType.UPDATE).pipe(
-    switchMap(action => {
+    withLatestFrom(this.store$),
```

Comment:
Would not it be better to use selector here instead in order to get just the organizations?

Reply:
Again, good catch :)))